autoscheduler/scheduler/views.py

```diff
@@ -83,14 +85,13 @@ def post(self, request):
                              for avail in query["availabilities"]]
 
         term = query["term"]
-        include_full = query["includeFull"]
 
         num_schedules = 5
 
         schedules = []
         message = ''
         try:
-            schedules = create_schedules(courses, term, unavailable_times, include_full,
+            schedules = create_schedules(courses, term, unavailable_times,
                                          num_schedules)

```

Comment:
```suggestion
            schedules = create_schedules(courses, term, unavailable_times, num_schedules)
```
This fits on one line now, hooray